hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -314,7 +314,8 @@ do
 done
 
 
-export ENABLE_ADMISSION_CONTROLLER=true
+export ADMISSION_CONTROLLERS__ENABLED=true
```

Comment:
Could you elaborate on what level of backwards compatibility we need to maintain here? If I try not to break backwards compatibility I will end up with one of the following:
* Scattered values in Helm Chart related to admission controllers
* Inconsistency in admission controller related environment variables
* Multiple env variables handling the same thing 
  * And then the next question arises, who is going to remove those and when is the right time to remove them since we don't have any releases to say e.g. after 3 releases we remove those deprecated env variables?

With this PR, I try to bring some structure on the certificate story for the admission controllers and to do so, I needed to adjust a couple of things for consistency. I think I have done sufficient work to indicate these changes. There are at least 2 ways to deploy the stack based on what I found:
* `make deploy-setup-k8s`
* Helm Chart

For the former, the docs are updated. Even before if someone would set the `ENABLE_ADMISSION_CONTROLLER` env variable they would need to dive deeper to understand which secrets they should pre-populate if they didn't follow the docs. Actually, they would notice because the webhooks pods couldn't start by manually tinkering the cluster. Although I don't like this behavior, I kept it the same to not break your usual flow.

For the latter, to our knowledge we are the only consumers of the Helm Chart and we will handle that appropriately. If the chart was versioned, we would bump the major version to indicate a breaking change and then the admin would have to be careful.

Reply:
My main concern is changing the name of the var ENABLE_ADMISSION_CONTROLLER. Our downstream version (openshift) uses OLM to install this operator and we would have to make some changes there as far as I see it (we have additional files). So, I need to understand what this implies to us. @SchSeba @zeeke, what do you think?

I'm still trying to understand how we deploy the certificates in openshift. Need to dig on that further.

Reply:
OpenShift deployment doesn't rely on `WEBHOOK_CA_BUNDLE` var
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/bindata/manifests/webhook/003-webhook.yaml#L8

Regarding the `ENABLE_ADMISSION_CONTROLLER`, we'll have to adjust the downstream deployment files, as you correctly stated:
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/7c2f7798f8e8f211ce40baee819bc00532c78ca7/bundle/manifests/sriov-network-operator.clusterserviceversion.yaml#L348
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/fbc5bc1a1948bf1b7be774156050c323fd70e5e6/config/manager/manager.yaml#L75
https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ed5bef6c658f0951ab57f49f8429622bf3ac87e1/config/manifests/bases/sriov-network-operator.clusterserviceversion.yaml#L130

I'm ok with the new name `ADMISSION_CONTROLLERS_ENABLED`

Reply:
Folks let me know if there is something for me to address here. If there is, please give me the branch or tag I should be doing the changes.